base/ltoutput.dtx

```diff
@@ -30,7 +30,7 @@
 %%% From File: ltoutput.dtx
 %<flafter>\ProvidesPackage{flafter}
 %<fltrace>\ProvidesPackage{fltrace}
-%<flafter,fltrace>       [2021/07/31 v1.4e
+%<flafter,fltrace>       [2022/06/23 v1.4f
```

Comment:
this looks wrong. Why should those register affect the sty files? I think you update the wrong date here (not that it matters much)